docs/action_kit.rst

```diff
@@ -5,19 +5,53 @@ ActionKit
 Overview
 ********
 
-The ActionKit class allows you to interact with an `ActionKit <https://actionkit.com/>`_.
+`ActionKit <https://actionkit.com/>`_ is a platform for advocacy, fundraising, and
+get-out-the-vote. This parsons integration with the
```

Comment:
One small thing: we capitalize "Parsons" when referencing the name of the library in documentation.